public/docs/_examples/structural-directives/ts/src/app/app.component.html

```diff
@@ -152,11 +152,9 @@ <h2 id="ngSwitch">NgSwitch</h2>
 
 <div>Pick your favorite hero</div>
 <p>
-  <ng-container *ngFor="let h of heroes">
-    <label>
-      <input type="radio" name="heroes" [(ngModel)]="hero" [value]="h">{{h.name}}
-    </label>
-  </ng-container>
+  <label *ngFor="let h of heroes">
```

Comment:
It is not an example of `<ng-container>`. This code is not shown in the page, it is only there to support the `ngSwitch` code which is shown in the page.